packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -93,13 +93,22 @@ module.exports = {
     }
   },
 
-  // Truffle DB is currently disabled by default; to enable it, change enabled: false to enabled: true
+  // Truffle DB is currently disabled by default; to enable it, change enabled:
+  // false to enabled: true, the default storage location can also be
+  // overridden by specifying the adapter settings.
   //
-  // Note: if you migrated your contracts prior to enabling this field in your Truffle project and want
-  // those previously migrated contracts available in the .db directory, you will need to run the following:
+  // Note: your previously migrated contracts will not be available in
+  // truffle-db. We recommnd you backup your artifacts before utilizing Truffle
+  // DB the first time with:
   // $ truffle migrate --reset --compile-all
 
   db: {
     enabled: false
+    //, adapter: {
```

Comment:
```suggestion
    // adapter: {
```

Reply:
The comma is needed to separate `enabled: false` which  would be needed if the adapter setting is un-commented